.github/pr-comments/for-community.md

```diff
@@ -0,0 +1,12 @@
+:smiley_cat: Thank you for creating this PR!
+
+Looks like you either do not have permissions to add labels in the `docs-site` repo, or you are using a fork in your PR. To publish your content to [Zowe Docs](https://docs.zowe.org/), follow these steps.
+
+- [ ] Enter the `/labels` command in a comment to add required `review`, `area`, and `release` labels to your pull request.
```

Comment:
Perhaps refer to `review`, `area`, and `release` as categories or label categories instead of labels...